lib/aws/s3.js

```diff
@@ -0,0 +1,25 @@
+import { S3 } from 'aws-sdk';
+
+class AwsS3 {
+  constructor() {
+    this._s3 = new S3({ region: 'eu-west-1' });
+  }
+
+  getSignedUrl(operation, params = {}) {
+    return this._s3.getSignedUrlPromise(operation, params);
+  }
+
+  getObject(params = {}) {
+    return this._s3.getObject(params);
+  }
+
+  copyObject(params = {}) {
+    return this._s3.copyObject(params).promise();
+  }
+
+  deleteObject(params = {}) {
+    return this._s3.deleteObject(params).promise();
+  }
+}
+
+export default new AwsS3();
```

Comment:
what is the motivation to have your own wrapper around aws-sdk?

Reply:
@sleepwalky I always try to encapsulate work with third-party in single file. If I suddenly decide to replace a third-party dependency with another module, I will only have to modify the code in 1 place, adjusting methods' implementation, whereas lib's interface will remain intact